lib/swarm/tracker/tracker.ex

```diff
@@ -1214,7 +1214,7 @@ defmodule Swarm.Tracker do
         current_node = Node.self
         case meta do
           %{mfa: _} when pid_node == current_node ->
-            # This was created via register_name/4, which means we need to kill the pid we started
+            # This was created via register_name/5, which means we need to kill the pid we started
```

Comment:
Not sure if `register_name` is known as arity 4 or 5.

Reply:
We use `register_name/5` in the README, so that's preferable.